### PR TITLE
Improve server console formatting

### LINKED ISSUE #209: Improve Console output format
Only 2 CSS rules for a much better formatting:

`font-family: monospace;
    white-space: pre;`

Before

image

After

image

### PR DESCRIPTION
**Pull request by alexanderzeillinger**
Closes #209

### CODE REVIEW

File: resources/views/console/index.blade.php [hunk 1]
```diff
@@ -65,7 +65,7 @@
 
         <div class="space-y-3">
             <x-console-view id="console-output">
```
Comment (saeedvaziry) [inline resources/views/console/index.blade.php:67]: could we have this in the component itself? `resources/views/components/console-view.blade.php`

File: resources/views/console/index.blade.php [hunk 2]
```diff
@@ -65,7 +65,7 @@
...
-                <div class="w-full" x-text="output"></div>
+                <div class="w-full font-mono whitespace-pre" x-text="output"></div>
```
Comment (saeedvaziry) [inline resources/views/console/index.blade.php:68]: could we have this in the component itself? `resources/views/components/console-view.blade.php`

**Comment by alexanderzeillinger** [comment]
At this point it's faster if you do it yourself.

**Comment by saeedvaziry** [comment]
@alexanderzeillinger you were doing good tho. but sure I will handle it.

**Pull request closed without merging**